src/acm.cpp

```diff
@@ -488,6 +488,10 @@ bool ASN1_Codec::configure() {
 
     // confluent cloud integration
     std::string kafkaType = getEnvironmentVariable("KAFKA_TYPE");
+    if (kafkaType == "") {
```

Comment:
Is there any other kafka type other than local and confluent? If not, how about put an else to "  `if (kafkaType == "CONFLUENT")` saying if confluent is not yet, it will by default use local kafka broker?

Reply:
This has been refactored to improve clarity.